staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -301,3 +302,61 @@ type UserInfo struct {
 
 // ExtraValue masks the value so protobuf can generate
 type ExtraValue []string
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AuditConfiguration represents a cluster level audit configuration
+type AuditConfiguration struct {
+	metav1.TypeMeta
+	// +optional
+	metav1.ObjectMeta
+
+	// if undefined it will default to the statically configured cluster policy if available
+	// if neither exist the backend will fail
+	// +optional
+	Policy *Policy
```

Comment:
why no spec?

/cc @liggitt

Reply:
This was agreed upon in the KEP https://github.com/kubernetes/community/blob/master/keps/sig-auth/0014-dynamic-audit-configuration.md#cluster-scoped-configuration but if we can add a spec if needed

Reply:
Went ahead and and added the spec here